LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2LWAbilitiesModTemplate.uc

```diff
@@ -32,6 +32,8 @@ static function UpdateAbilities(X2AbilityTemplate Template, int Difficulty)
     {
         Template.AbilityToHitCalc.OverrideFinalHitChanceFns.AddItem(OverrideFinalHitChance);
     }
+	if(Template.DataName='BondmateTeamwork'||Template.DataName=='BondmateTeamwork_Improved')
```

Comment:
`=` -> `==`. And since we're here, would you mind adding a space between `if` and `(`? Actually, spaces around operators too. I know the existing base is a mess, but I'd like to standardise moving forward. And it satisfies my OCD-like desire for consistent style.